Documentation/ceph-cluster-crd.md

```diff
@@ -777,7 +777,7 @@ spec:
     useAllDevices: true
     deviceFilter:
     config:
-      metadataDevice:
+      metadataDevice: "/dev/disk/by-id/ata-ST4000DM004-XXXX" # metadataDevice can be specified using full udev path
```

Comment:
I'd suggest we not add the example here, we want to keep this first example for simply creating OSDs on all devices (and no metadataDevice). It seems sufficient that you added it already to the paragraph describing metadataDevice.